packages/tools/guiEditor/src/components/propertyTab/propertyTabComponent.tsx

```diff
@@ -115,6 +120,22 @@ export class PropertyTabComponent extends React.Component<IPropertyTabComponentP
         );
     }
 
+    /**
+     * Read loaded file
+     * @param file 
+     */
+    loadContorl(file: File) {

```

Comment:
Another contorl => control.

You should perform a global search & replace to replace all occurrences of `contorl` with `control`.